content/docs/setup/kubernetes/prepare/platform-setup/ibm/index.md

```diff
@@ -20,27 +20,27 @@ To install the managed Istio add-on in IBM Cloud Public, see the [IBM Cloud Kube
 
 ## IBM Cloud Public
 
-Replace `<cluster-name>` with the name of the cluster you want to use in the following instructions.
+1. [Install the IBM Cloud CLI, the IBM Cloud Kubernetes Service plug-in, and the Kubernetes CLI](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install).
 
-1. Create a new lite or paid Kubernetes cluster.
+1. Create a billable Kubernetes cluster. Replace `<cluster-name>` with the name of the cluster you want to use in the following instructions.
 
-    Lite cluster:
+    {{< tip >}}
+    To see available zones, run 'ibmcloud ks zone-ls'.
```

Comment:
should we mention region info too so they can select a region then zone?

Reply:
@linsun I don't know that I want to go into regions with the base `ibmcloud` CLI or maybe you mean the `ibmcloud ks region *` commands. I believe from the docs and the CLI help, that _zone_ is all that is used within the IKS CLI during cluster creation.

Reply:
if the zone they are specifying isn't in the default region that is being targeted, there will be error somewhere.  that is why I suggested adding it since we ask users to specify zone info.

cc @Rachael-Graham

Reply:
I believe the _region_ is get/set/unset with `ibmcloud ks region get/set/unset` and one uses the `ibmcloud ks zone ls` to get the zones that are acceptable for --zone <name> on the actual `ibmcloud ks cluster create` based on the currently set region. There is no --region on the command.

That said: We could extend the current zone tip:

  {{< tip >}}
    To see available zones, run 'ibmcloud ks zone-ls'. Zones are isolated from each other, which ensures no shared single point of failure. [IBM Cloud Kubernetes services zones and regions](https://cloud.ibm.com/docs/containers?topic=containers-regions-and-zones#regions-and-zones) describes regions, zones, and how to specify the region and zone for your new cluster.
    {{< /tip >}}

Let me know if there is better wording, or if this should added.

Reply:
I like that, so users can select a valid zone based on whatever region they are already configured. :). thanks Eric.

Reply:
@linsun Your command and my command provide the same output. It looks like the command you reference is what is on the doc referenced in the tip while the command I showed is what is in the `ibmcloud ks cluster create` help text:

```       --zone value            Specify the zone where you want to create the cluster, the options depend on what region that you are logged in to. To see available zones, run 'ibmcloud ks zone-ls'. For IBM Cloud Dedicated accounts, do not specify this option because the zone is preselected based on where your IBM Cloud infrastructure account is located.```

Reply:
Changes I mentioned above are pushed. Still waiting on @Rachael-Graham.

Reply:
Yep, y'all have it right--should only mention zone. (Regions will be going away soon anyway, so this is good also to emphasize zone instead of region).